baybe/utils/chemistry.py

```diff
@@ -12,17 +13,15 @@
 from joblib import Memory
 
 from baybe._optional.chem import (
-    Calculator,
+    BaseFingerprintTransformer,
```

Comment:
Why are these imports made via the `_optional.chem` subpackage? Is it to cluster all of the chemistry related parts in a single file and thus avoid problems if e.g. `skfp` is not installed or is there another reason why we not simply do e.g. `from skfp import BaseFingerPrintTransformer` here?

Reply:
not sure if I understand your question
but I think this is how optional imports are supposed to be handled, no?

Reply:
True that.